src/BlazorWebView/src/Maui/iOS/iOSMauiAssetFileProvider.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using Foundation;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.Components.WebView.Maui
+{
+	/// <summary>
+	/// A minimal implementation of an IFileProvider to be used by the BlazorWebView and WebViewManager types.
+	/// </summary>
+	internal sealed class iOSMauiAssetFileProvider : IFileProvider
```

Comment:
```suggestion
	internal sealed class IOSMauiAssetFileProvider : IFileProvider
```

So it is similar to `IOSWebViewManager`?

Reply:
@campersau this file is for serving static assets such as index.html, CSS, PNG, JS, etc. within the BlazorWebView. The WebViewManager calls into this file provider to get the contents of the files.